File: Number Series/Java/FindMin.java

import java.util.*;

class Program {
	public static void main(String args[]) {
		Scanner sc = new Scanner(System.in);
		int num1, num2, num3;
		System.out.print("Enter three numbers: ");
		num1 = sc.nextInt();
		num2 = sc.nextInt();
		num3 = sc.nextInt();

		if(num1 < num2) {
			if(num1 < num3)
				System.out.print(num1);
			else	
				System.out.print(num3);
		}
		else {
			if(num2 < num3)
				System.out.print(num2);
			else	
				System.out.print(num3);

		}
		System.out.println(" is a minimum among " + num1 + " " + num2 + " and "+ num3);
	}
}

File: Number Series/C/FindMin.c
/*
 * Program 5: Write a Program that accepts Three integers from user and prints
minimum number from them.
Input: 56 7 99
Output: The Minimum number amongst 56 7 & 99 is 7
*/

#include <stdio.h>

void main() {
	int num1,num2,num3;
	printf("Enter three numbers: ");
	scanf("%d%d%d" , &num1,&num2,&num3);

	int min = num1 < num2 ? ( num1 < num3 ? num1 : num3) : (num2 < num3 ? num2 : num3 );
      	printf("The minimum number among %d %d and %d is %d\n",num1,num2,num3, min);	


}
